keyboards/handwired/asesmero/2x6/keymaps/via/readme.md

```diff
@@ -0,0 +1,49 @@
+
+LAYER 0 (RED)
++------------+------------+------------+------------+------------+------------+
+|    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |
+| + Shift    | + Shift    | + Shift    | + Shift    | + Shift    | + Shift    |
+| + Alt + A  | + Alt + B  | + Alt + C  | + Alt + D  | + Alt + E  | + Alt + F  |
++------------+------------+------------+------------+------------+------------+
+|    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |    Ctrl    |     TO     |
+| + Shift    | + Shift    | + Shift    | + Shift    | + Shift    |    LAYER   |
+| + Alt + G  | + Alt + H  | + Alt + I  | + Alt + J  | + Alt + K  |  1 (GREEN) |
++------------+------------+------------+------------+------------+------------+
+
+LAYER 1 (GREEN)
++------------+------------+------------+------------+------------+------------+
+|            |            |            |            |            |            |
+|TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |
+|            |            |            |            |            |            |
++------------+------------+------------+------------+------------+------------+
+|            |            |            |            |            |     TO     |
+|TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |    LAYER   |
+|            |            |            |            |            |  2 (BLUE)  |
++------------+------------+------------+------------+------------+------------+
+
+LAYER 2 (BLUE)
++------------+------------+------------+------------+------------+------------+
+|            |            |            |            |            |            |
+|TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |
+|            |            |            |            |            |            |
++------------+------------+------------+------------+------------+------------+
+|            |            |            |            |            |     TO     |
+|TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |    LAYER   |
+|            |            |            |            |            | 3 (PURPLE) |
++------------+------------+------------+------------+------------+------------+
+
+LAYER 3 (PURPLE)
++------------+------------+------------+------------+------------+------------+
+|            |            |            |            |            |            |
+|TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |
+|            |            |            |            |            |            |
++------------+------------+------------+------------+------------+------------+
+|            |            |            |            |            |     TO     |
+|TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |TRANSPARENT |    LAYER   |
+|            |            |            |            |            |  0 (RED)   |
++------------+------------+------------+------------+------------+------------+
```

Comment:
Same again with the triple backtick sequence \`\`\`; before Line 7 and again at the end of the file.